frontend/Dockerfile

```diff
@@ -51,7 +51,10 @@ ENV ADD_HEADER_HSTS="" \
     MIXPANEL_API_URL=${MIXPANEL_API_URL}
 
 COPY ./frontend/nginx.conf.template /etc/nginx/templates/default.conf.template
+COPY ./frontend/nginx.conf /etc/nginx/nginx.conf
+RUN chmod -R a+w /etc/nginx/conf.d
```

Comment:
instead of giving rights to all can't we use the user ID and add a `USER` statement here?